docs/note.md

```diff
@@ -266,7 +265,71 @@ Instead use a video div with two sources and a poster that gets loaded as a prev
 
 {{<video webm_src="../img/heart.webm" mp4_src="../img/heart.mp4" alt="A robot drawing a heart" poster="../img/heart.jpg">}}
 
-or if you want a video without controls - mimicking a GIF:
+We use `webm` and `mp4` source files for videos because they are generally smaller.
+The poster is an image that gets loaded as a preview.
+
+To create the `webm` and mp4 files use these commands:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -i PATH_TO_GIF_OR_VID -vcodec libx264 -vf "format=yuv420p,scale=720:-1" -b:v 400k PATH_TO_GIF_OR_VID.mp4
+ffmpeg -i PATH_TO_GIF_OR_VID -c vp9 -b:v 0 -crf 41 my-animation.webm
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -i PATH_TO_GIF_OR_VID -vcodec libx264 -vf "format=yuv420p,scale=720:-1" -b:v 400k PATH_TO_GIF_OR_VID.mp4
+ffmpeg -i PATH_TO_GIF_OR_VID -c:v libvpx-vp9 -b:v 0 -crf 41 my-animation.webm
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The first command:
+
+- uses the `libx264` codec
+- uses the `yuv420p` pixel format
+- scales the video to `720px` width
+- changes the bitrate to `300k` - you can change this value but check that the result is usable and reasonably small
+
+The second command:
+
+- uses the `vp9` codec
+- `-crf 41` sets the quality level.
+  Valid values are 0-63.
+  Lower numbers are higher quality
+
+To create a preview image use this command:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -ss 00:00:05 -i PATH_TO_GIF_OR_VID.mp4 -frames:v 1 PATH_TO_GIF_OR_VID.jpg
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ffmpeg -ss 00:00:05 -i PATH_TO_GIF_OR_VID.mp4 -frames:v 1 PATH_TO_GIF_OR_VID.jpg
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The command takes a screenshot at `00:00:05`.
+
+### GIF-like videos
+
+GIF-like videos on our pages are generally used to show robot actions.
+We do not use the GIF file format because they use a lot of bandwidth - more than videos - and the [best practice](https://developer.chrome.com/en/docs/lighthouse/performance/efficient-animated-content/) is to not use them.
```

Comment:
```suggestion
We do not use the GIF file format because it uses a lot of bandwidth - more than videos - and the [best practice](https://developer.chrome.com/en/docs/lighthouse/performance/efficient-animated-content/) is to not use them.
```